### ISSUE TITLE
Crate name should not include -rs

### ISSUE THREAD

**Issue by chris-morgan** [sciter-sdk/rust-sciter#41]
The crate name should be `sciter`, not `sciter-rs`. Crate names should not include a `rust-` prefix or `-rs` suffix, because we already know it’s Rust.

**Comment by pravic** [comment]
Hi.

The intention was to show that this crate is a bindings library. During that time it was [somewhat common](https://crates.io/search?q=-rs) (and `-sys` crates were not so popular). Nowadays it would be better to split it to a `-sys` crate and a library with high level Rust abstractions over the former. 

May I ask why it **should not** include?

**Comment by chris-morgan** [comment]
-rs has never been *popular* and was already considered an error when crates.io was first released. Unfortunately, that opinion was not overtly stated anywhere in documentation or warnings inside Cargo or on uploading to crates.io—it was merely held in IRC channels and such—and so some people went and did it. The most obvious explanation at the time seemed to be that people’s *repositories* often had a rust- prefix or .rs or -rs suffix, which was quite understandable (you do often want to indicate the language in the repository name for bindings), and it grew from there.

The -sys convention did arrive a little later; and yes, splitting this repository into two crates sciter-sys and sciter would be a more idiomatic approach now.

Packages and crates should not include these prefixes and suffixes, because the very fact that you’re using them indicates it’s already Rust. rust-, -rs, _rs—these add nothing of semantic meaning, and only make it harder to find it: it makes sense that the canonical crate for Sciter should be called `sciter`, right?

**Comment by pravic** [comment]
I partly agree. But again, what if my library isn't canonical? What if someone will make a better library? He'll deserve the `crates.rs/sciter` URL (J.)

Anyway, it is indeed something to think about during the future improvements.

**Comment by chris-morgan** [comment]
Yeah, there has never been a good answer for *that* question, in any package manager. Sometimes you can pass the name off to someone else, but far too often in such cases the original user of the name is no longer around…
